theme_5/scss/_modules.scss

```diff
@@ -0,0 +1,221 @@
+}
+
+.menu-button {
+	background-color: $cerise-red;
+	border: 1px solid $cerise-red;
+	display: inline-block;
+	padding: .5em;
+	&.is-active {
+		.burger-icon {
+			@include burger-to-cross;
+		}
+	}
+}
+
+.badge-conf{
+	color: $white-solid;
+	line-height: 50px;
+	position: absolute;
+	right: 50px;
+}
+
+.logo {
+  background: url(../assets/logos_png/logo1_fiorella_mobil.png) no-repeat center;
+  height: 50px;
+}
+
+// Sprites
+.cart-icon {
+	background: url(../assets/Sprite.png) no-repeat -171px -383px;
+	height: 19px;
+	line-height: 50px;
+	position: absolute;
+	top: 15px;
+	width: 20px;
+
+}
+
+.bus-icon {
+	background: url(../assets/Sprite.png) no-repeat -224px -48px;
+	width: 68px;
+	height: 52px;
+	transform: scale(.7);
+}
+// Input
+.container-text {
+	padding: 7px 7px;
+	margin: 0;
+	.text-box {
+		background: url(../assets/Sprite.png) no-repeat 230px -50px;
+		box-shadow: none;
+		display: inline-block;
+		border: none;
+		border-bottom: 1px solid $cerise-red;
+		font-family: $merriweather-font;
+		font-size: 13px;
+		font-style: normal;
+		margin: 0 auto;
+		outline: none;
+		padding: 1px 0px 5px 0px;
+		text-transform: uppercase;
+	}
+
+	// @media (min-width: $mobile-portrait) {
+	// 	background: url(../assets/Sprite.png) no-repeat 550px -50px;
+	// }
+}
+
+// Carousel
+.carousel {
+	background-image: url(../assets/home/mobile/hero1_home_mobil.jpg);
+	background-size: cover;
+	background-position: center;
+	height: 50%;
+	position: relative;
+}
+
+.carousel-content {
+	position: absolute;
+	right: 10px;
+	top: 190px;
+}
+
+.carousel-content-title {
+	text-align: right;
+}
+
+// Articles
+.article {
+	background-color: $peppermint;
+	height: 495px;
+	text-align: center
+}
+
+.article-title {
+	font-family: $playfair-font;
+	overflow: hidden;
+	text-align: center;
+}
+
+.article-content {
+	color: $emperor;
+	padding: 10px 20px;
+	p {
+		line-height: 30px;
+	}
+}
+
+// Collection
+.collection-item {
+	position: relative;
+	text-align: center;
+	margin-bottom: 10px;
+}
+
+.collection-link {
+	color: $white-solid;
+	font-family: $playfair-font;
+	font-size: 25px;
+	font-weight: 700;
+	position: absolute;
+	text-transform: uppercase;
+	top: 50%;
+	transform: translate(-50%, -50%);
+	width: 100%;
+	height: 100%;
+	padding-top: 25%;
+	&:hover {
+		color: $white-solid;
+	}
+	&:active {
+		color: $white-solid;
+	}
+}
+
+// Promo
+.promo {
+	background-image: url(../assets/home/mobile/banner_image_home_mobil.jpg);
+	background-size: cover;
+}
+
+.promo-title {
+	overflow: hidden;
+	text-align: left;
+}
+
+// Headings
+.h-style {
+	display: inline-block;
+	font-style: normal;
+	padding: 25px 15px;
+	position: relative;
+	text-transform: uppercase;
+	&:before {
+		background: $cerise-red;
+	  content: "";
+	  display: block;
+	  height: 1px;
+	  position: absolute;
+    top: 50%;
+	  width: 50%;
+		right: 100%;
+	}
+	&:after {
+		background: $cerise-red;
+	  content: "";
+	  display: block;
+	  height: 1px;
+	  position: absolute;
```

Comment:
Properties should be ordered background, content, display, height, left, position, top, width